grow/commands/subcommands/download_translations.py

```diff
@@ -16,7 +16,7 @@
 @shared.locale_option(help_text='Which locale(s) to download. If unspecified,'
                                 ' translations for all locales will be downloaded.')
 @shared.service_option
-@shared.include_obsolete_option(CFG, default_value=True)
+@shared.include_obsolete_option(CFG, default_value=False)
```

Comment:
Also, the default `default_value` is False :)

Reply:
Thanks, removed the unnecessary code. :)